src/node/rpc/test/membervoting_test.cpp

```diff
@@ -209,14 +207,14 @@ TEST_CASE("Member query/read")
     Store::Tx tx;
     tx.get_view(network.whitelists)
       ->put(WlIds::MEMBER_CAN_READ, {Tables::VALUES});
-    CHECK(tx.commit() == kv::CommitSuccess::OK);
+    REQUIRE(tx.commit() == kv::CommitSuccess::OK);
 
     bool compile = true;
     do
     {
       const auto req = create_json_req(query_params(query, compile), "query");
-      const Response<int> r = frontend_process(frontend, req, member_cert);
-      CHECK(r.result == value);
+      const Response<int> r = frontend_process(frontend, req, member_caller);
```

Comment:
`membervoting_test.cpp` is a very large diff, but I think we can remove most of the changes? The `CHECK`s that have become `REQUIRE`s, is there a strong reason for this?

And if `frontend_process` took the PEM cert and did `const auto der = tls::make_verifier(pem)->der_cert_data();` internally, we could drastically reduce the number of changes in this file. The changes from `member_cert` to `member_caller` would revert to passing the PEM `member_cert`.

Reply:
You can tell it happened on a Friday.